### PR TITLE
fix(RHICOMPL-1005): Address issues with more spaces in filter labels

### PR DESCRIPTION
**Pull request by bastilian**
Fixes the issues mentioned in https://github.com/RedHatInsights/compliance-frontend/blob/master/src/PresentationalComponents/ReportsTable/Filters.js#L45

### CODE REVIEW

File: packages/inventory-compliance/src/Utilities/FilterConfigBuilder.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import FilterBuilder from './FilterBuilder';
 import ChipBuilder from './ChipBuilder';
 
 export const stringToId = (string) => (
-    string.replace(' ', '').toLowerCase()
+    string.split(' ').join('').toLowerCase()
```
Comment (vkrizan) [inline packages/inventory-compliance/src/Utilities/FilterConfigBuilder.js:6]: Nitpick: `.replace(/ /g, '')`

**Review by vkrizan — APPROVED** [review]
Works!

File: packages/inventory-compliance/src/Utilities/FilterConfigBuilder.js (hunk 1 above)
Comment (bastilian) [reply packages/inventory-compliance/src/Utilities/FilterConfigBuilder.js:6]: Why?

Comment (vkrizan) [reply packages/inventory-compliance/src/Utilities/FilterConfigBuilder.js:6]: Does the same with one call.

**Pull request merged**